src/validate.ts

```diff
@@ -51,21 +51,21 @@ interface ExecuteRequest{
      payload: 
      { devices: 
         { id: string; 
-        customData: customDatas;}[];
+        CustomData: CustomData;}[];
     execution: { command: string; params: params; }[];
     }; 
     }[];
 }
 
-export function generateExecuteRequest(deviceIds: string[], customDatas: customDatas[], commands: string[], params: params[]): ExecuteRequest{
+export function generateExecuteRequest(deviceIds: string[], CustomData: CustomData[], commands: string[], params: params[]): ExecuteRequest{
```

Comment:
Is there a type called `params` in your code?

Reply:
There is, I'm not sure why that didn't commit so I will push it again.